org.adempiere.base/src/org/compiere/util/DefaultEvaluatee.java

```diff
@@ -175,7 +175,7 @@ public String get_ValueAsString (Properties ctx, String variableName)
 		boolean globalVariable = Env.isGlobalVariable(variableName);
 		boolean tabOnly = m_onlyTab != null ? m_onlyTab.booleanValue() : false;
 		// get value from window context or global
-		if (m_windowNo >= 0)
+		if (m_windowNo != 0)
```

Comment:
with this changes, we should change the DefaultEvaluatee constructor that initiate m_windowNo to -1, to 0 instead.

Reply:
> with this changes, we should change the DefaultEvaluatee constructor that initiate m_windowNo to -1, to 0 instead.

Thanks @hengsin - committed - all 490 unit tests passed